_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+published: true
+title: 'Solidity Core Team Updates'
+date: '2022-12-05'
+author: Solidity Team
+category: Announcements
+---
+
+More than two years have passed since we [introduced Solidity core team members](https://blog.soliditylang.org/2020/09/18/meet-the-team/) on the blog and we realized it is high time for some updates: Meet new team members, find out who moved on to other adventures and learn about recent changes in the team structure!
+
+Before we dive in, a reminder that the Solidity programming language and compiler are **open-source community projects**. This post dives into the core team that leads the development. Nevertheless, we cannot stress enough how invaluable community contributions and interactions are for Solidity. We value feedback and input and want to thank our community for the continued support!
+
+## Thank you to our alumni 💐
+
+Like in any other project, there naturally comes a time for some team members to move on to other challenges. Since our last update, we have bid farewell to Christian (@christianparpart), Damian (@wechman), Đorđe (@mijovic), Hari (@hrkrshnn), Mathias (@marenz),  and Nishant (@nishant-sachdeva). We were extremely glad to have them and wish them all the best for their future! Luckily, some alumni remain in our orbit and continue to contribute with code and input to language design discussions.
```

Comment:
```suggestion
Like in any other project, there naturally comes a time for some team members to move on to other challenges. Since our last update, we have bid farewell to Christian (@christianparpart), Damian (@wechman), Đorđe (@mijovic), Hari (@hrkrshnn), Mathias (@marenz),  and Nishant (@nishant-sachdeva). We were delighted to have them and wish them all the best for their future! Luckily, some alumni remain in our orbit and continue contributing with code and input to language design discussions.
```